examples/firebase/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "functions",
+  "description": "Cloud Functions for Firebase",
+  "scripts": {
+    "serve": "firebase emulators:start --only functions",
```

Comment:
Could you please add these scripts?

```
"service:link-pg": "cd ../.. && npm run build && npm link",
"preinstall": "npm run clear:deps",
"postinstall": "npm run service:link-pg && npm link passkit-generator",
"clear:deps": "rm -rf node_modules",
```

So that we can link the local passkit-generator version when installing.